builtin/logical/pki/path_acme_order.go

```diff
@@ -268,6 +268,10 @@ func (b *backend) acmeFinalizeOrderHandler(ac *acmeContext, _ *logical.Request,
 		return nil, err
 	}
 
+	if err := b.acmeState.TrackIssuedCert(ac, order.AccountId, hyphenSerialNumber, order.OrderId); err != nil {
+		return nil, err
```

Comment:
hmm we have the same issue of orphaning a certificate if we error out here as we did below, and I never got around to trying to delete it. Should we perhaps combine these two operations with a single error handler/warning message?

Reply:
I've re-added the warning!

Reply:
Thanks!